sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.cs

```diff
@@ -98,6 +104,12 @@ public class TestConfigurations
         public static TenantConfiguration DefaultTargetOAuthTenant =>
             GetTenant("TargetOAuthTenant", s_configurations.Value.TargetOAuthTenantName);
 
+        /// <summary>
+        /// Gets a tenant to use for any tests that require hierarchical namespace
+        /// </summary>
+        public static TenantConfiguration DefaultNamespaceEnabledTenant =>
```

Comment:
Nit - can we make the name match TargetHierarchicalNamespaceTenant since there are so many of these?

Reply:
We need a separate tenant for hierarchical namespace - there are blob features, such as blob snapshot - that don't function with hierarchical namespace enabled.